packages/nimble-components/src/table-column/base/models/column-internals.ts

```diff
@@ -10,8 +10,11 @@ import {
 import type { TableGroupRow } from '../../../table/components/group-row';
 import { createGroupHeaderViewTemplate } from '../group-header-view/template';
 import { createCellViewTemplate } from '../cell-view/template';
+import { ColumnValidator } from './column-validator';
 
-export interface ColumnInternalsOptions {
+export interface ColumnInternalsOptions<
```

Comment:
MentionInternals is following a similar pattern to columnInternals. In a follow-up PR the updated validator patterns should try and align there as well (or be captured in a tech debt if for some reason that is not relatively straightforward).

Reply:
I'll take a look.